it/src/main/java/com/google/cloud/teleport/it/bigtable/StaticBigtableResourceManager.java

```diff
@@ -0,0 +1,284 @@
+/*
+ * Copyright (C) 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package com.google.cloud.teleport.it.bigtable;
+
+import static com.google.cloud.teleport.it.bigtable.BigtableResourceManagerUtils.checkValidTableId;
+import static com.google.cloud.teleport.it.common.ResourceManagerUtils.checkValidProjectId;
+
+import com.google.api.gax.core.CredentialsProvider;
+import com.google.api.gax.rpc.ServerStream;
+import com.google.cloud.bigtable.admin.v2.BigtableInstanceAdminSettings;
+import com.google.cloud.bigtable.admin.v2.BigtableTableAdminClient;
+import com.google.cloud.bigtable.admin.v2.BigtableTableAdminSettings;
+import com.google.cloud.bigtable.data.v2.BigtableDataClient;
+import com.google.cloud.bigtable.data.v2.BigtableDataSettings;
+import com.google.cloud.bigtable.data.v2.models.Query;
+import com.google.cloud.bigtable.data.v2.models.Row;
+import com.google.cloud.bigtable.data.v2.models.RowMutation;
+import com.google.common.base.Strings;
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.Iterables;
+import java.io.IOException;
+import java.util.List;
+import java.util.Objects;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.threeten.bp.Duration;
+
+/**
+ * An implementation for implementation of {@link BigtableResourceManager} interface which allows to
+ * run test as if new resources are created, but reuse pre-created resources all the time.
+ *
+ * <p>The class is thread-safe.
+ */
+public class StaticBigtableResourceManager implements BigtableResourceManager {
```

Comment:
Are we planning on checking this in or is this just a placeholder until we have change stream support in the admin client?

Reply:
This is a placeholder until Cdc can be enabled from Admin client.

Reply:
Will remove this before merging

Reply:
After talking to @Polber , we will remove IT for now until CDC GA and then we can spin up hermetic bigtables with CDC enabled